readme.md

```diff
@@ -40,124 +40,139 @@ $ npm -v
 
 - [ ] Run `npm init -y` from the command line
 
 * **Question:** What does `npm init` do? How does the `-y` flag modify that command?
 
 * **Your Answer:**
 
----
+---npm init command initializes the project and creates repository, bugs and homepage in the package.json file. -y flag takes the default project name and settings and does not ask it.
 
 - [ ] Take a look at the file that was generated by the previous command
 
 * **Question:** What is the purpose of the following keys? "name", "scripts", "license"
 
 * **Your Answer:**
 
----
+---name - contains package name
+scripts - dictionary containing script commands that are run at various times in the lifecycle of your package. The key is the lifecycle event, and the value is the command to run at that point. By default, creates an empty test script
+license - license of the package so that users know how to use the package
 
 - [ ] Create a `.gitignore` file
 
 * **Question:** What is the purpose of the `.gitignore` file? What is the significance of a "dot-file?"
 
 * **Your Answer:**
 
----
+--- gitignore file specifies intentionally untracked files that Git should ignore. dot-file are files treated as hidden files.
 
 - [ ] Create an `index.js` file with the following contents: `console.log('Hello, Node!')`
 
 * **Question:** From the command line, how can you run this file?
 
 * **Your Answer:**
 
----
+---node index.js
 
 - [ ] Run `npm test` from the command line
 
-* **Question:** What happens and how is this related to what is in the `package.json` file? 
+* **Question:** What happens and how is this related to what is in the `package.json` file?
 
 * **Your Answer:**
 
----
+---> w1-node-ecosystem@1.0.0 test C:\Rubini\Technical\Personal\UW\JScript400A\w1-node-ecosystem
+
+> echo "Error: no test specified" && exit 1
+
+"Error: no test specified"
+npm ERR! Test failed. See above for more details.
+
+test Scripts from package.json are run
 
 - [ ] Create a new "script" command called "start" that has the following value: `node index.js`
 
 * **Question:** What will you enter on the command line to run that script?
 
 * **Your Answer:**
 
----
+---npm start
 
 - [ ] Change the name of your "start" script to "my-file"
 
 * **Question:** The same pattern will not work to try and run this script. How can you successfully get this script to run?
 
 * **Your Answer:**
 
----
+---npm run my-file
 
 - [ ] Create a new file called `profile.js`. Inside the file, copy the following but replace `<your-name>` with your name:
+
   ```js
-  module.exports = '<your-name>'
+  module.exports = "<your-name>";
   ```
 
   Add the following to your `index.js` file. Then, run your file.
+
   ```js
-  const profile = require('./profile.js')
-  console.log(profile)
+  const profile = require("./profile.js");
+  console.log(profile);
   ```
 
 * **Question:** What gets logged? Why?
 
 * **Your Answer:**
+  Hello, Node!
+  Rubini
+  The file profile.js is logged in the console.
```

Comment:
To be more precise, whatever it is you've _exported_ gets logged. It's not necessarily the entire file. This becomes more important when you have multiple functions in a file.